zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4008,7 +4008,7 @@ components:
       properties:
         process:
           $ref: "#/components/schemas/DeploymentProcess"
-        decision:
+        decisionDefinition:
```

Comment:
🔧 If we go for `decisionDefinition` here, we should go for `processDefinition` above as well, I think.